OpenRA.Game/FileSystem/InstallShieldCABExtractor.cs

```diff
@@ -404,6 +404,16 @@ public string DirectoryName(uint index)
 			return test;
 		}
 
+		public uint GetIndexFromFilename(string directory, string filename)
+		{
+			var path = directory + "\\" + filename;
```

Comment:
I am not sure as this is inside a CAB file, but `\` is the Windows path separator. Hardcoding it will break everywhere else. I suggest you `Path.DirectorySeparatorChar` instead.